.github/workflows/cpp-python-tests.yml

```diff
@@ -7,102 +7,116 @@ on:
+        run: echo "::set-output name=version::$(cat VERSION)"
+        shell: bash
+
+      - name: Print Version Information
+        run: echo "Version ${{ steps.generate_version_output.outputs.version }}"
+        shell: bash
+
+      - name: Generate OS Release Output
+        id: generate_os_release_name_output
+        run: echo "::set-output name=os_release_name::$(lsb_release -i -s)"
+        shell: bash
+
+      - name: Generate OS Release Version Output
+        id: generate_os_release_version_output
+        run: echo "::set-output name=os_release_version::$(lsb_release -r -s)"
+        shell: bash
+
+      - name: Generate OS Release Architecture Output
+        id: generate_os_release_architecture_output
+        run: echo "::set-output name=os_release_architecture::$(uname -i)"
+        shell: bash
+
+      - name: Generate Release File Name
+        id: generate_release_file_name
+        run: |
+          echo "::set-output name=release_file_name::opentera-webrtc-native-client-${{runner.os}}-\
+          ${{ steps.generate_os_release_name_output.outputs.os_release_name }}-\
+          ${{ steps.generate_os_release_version_output.outputs.os_release_version }}-\
+          ${{ steps.generate_os_release_architecture_output.outputs.os_release_architecture }}-\
+          ${{ steps.generate_version_output.outputs.version }}"
+
+      - name: Print Release File Name
+        run: echo "Release File Name ${{ steps.generate_release_file_name.outputs.release_file_name }}"
+        shell: bash
+
+      - name: Install system dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install ninja-build cmake build-essential libssl-dev libboost-all-dev
+          sudo apt-get install libglib2.0-dev libgtk-3-dev libpulse-dev libasound2-dev tree
+      - name: Install python dependencies
+        run: |
+          python3 -m pip install --user --upgrade pip
+          python3 -m pip install --user --upgrade setuptools wheel
+          python3 -m pip install --user --upgrade twine
+      - name: Create build directory
+        working-directory: opentera-webrtc-native-client
+        run: |
+          mkdir build
+
+      - name: Run CMake and build opentera-webrtc-native-client
+        working-directory: opentera-webrtc-native-client/build
+        run: |
+          cmake ../ -DCMAKE_BUILD_TYPE=${{ matrix.build-type }}
+          make -j2
+          make install
+        # rm -rf ../dist/${{ matrix.build-type }}/opentera-webrtc-native-client-{{ steps.generate_version_output.outputs.version }}/opentera_webrtc_native_client/build
+        # rm -rf ../dist/${{ matrix.build-type }}/opentera-webrtc-native-client-{{ steps.generate_version_output.outputs.version }}/opentera_webrtc_native_client/*.so
+        # rm -rf ../dist/${{ matrix.build-type }}/opentera-webrtc-native-client-{{ steps.generate_version_output.outputs.version }}/opentera_webrtc_native_client/opentera_webrtc_native_client.egg-info
+
```

Comment:
Why the comments?

If this is not needed anymore, I would delete the lines.

Reply:
I'm in the process of trying to understand if the paths need to be changed.
Not sure why they were there in the first place, maybe they are not required.